src/current/_includes/releases/v24.2/v24.2.0.md

```diff
@@ -0,0 +1,144 @@
+    - [Additional resources](#v24-2-0-additional-resources)
+
+{{ site.data.alerts.callout_info }}
+In CockroachDB Self-Hosted, all available features are free to use unless their description specifies that an Enterprise license is required. For more information, refer to the [Licensing FAQ](https://www.cockroachlabs.com/docs/stable/licensing-faqs).
+{{ site.data.alerts.end }}
+
+<div id="feature-highlights">
+
+<h4 id="v24-2-0-change-data-capture">Change Data Capture</h4>
+
+{% include releases/v24.2/feature-highlights-change-data-capture.html %}
+
+<h4 id="v24-2-0-migrations">Migrations</h4>
+
+{% include releases/v24.2/feature-highlights-migrations.html %}
+
+<h4 id="v24-2-0-sql">SQL</h4>
+
+{% include releases/v24.2/feature-highlights-sql.html %}
+
+<h4 id="v24-2-0-operations">Operations</h4>
+
+{% include releases/v24.2/feature-highlights-operations.html %}
+
+<h4 id="v24-2-0-observability">Observability</h4>
+
+{% include releases/v24.2/feature-highlights-observability.html %}
+
+{% include releases/v24.2/feature-detail-key.html %}
+
+</div>
+
+<h4 id="v24-2-0-backward-incompatible-changes-and-deprecations">Backward-incompatible changes and deprecations</h4>
+
+CockroachDB v24.2.0 includes no backward-incompatible changes or deprecations.
+
+<h4 id="v24-2-0-key-cluster-setting-changes">Key Cluster Setting Changes</h4>
+
+The following changes should be reviewed prior to upgrading. Default cluster settings will be used unless you have manually set a value for a setting. This can be confirmed by running the SQL statement `SELECT * FROM system.settings` to view the non-default settings.
+
+- [Settings added](#v24-2-0-settings-added)
+- [Settings removed](#v24-2-0-settings-removed)
+- [Settings with changed defaults](#v24-2-0-settings-with-changed-defaults)
+- [Settings with new options](#v24-2-0-settings-with-new-options)
+- [Settings with new aliases](#v24-2-0-settings-with-new-aliases)
+
+<h5 id="v24-2-0-settings-added">Settings added</h5>
+
+- `debug.zip.redact_addresses`: Added the [cluster setting]({% link v24.2/cluster-settings.md %}) `debug.zip.redact_addresses.enabled` that allows the user to enable or disable redaction of fields like `hostname` and IP addresses. [#123544][#123544] |
+- `kv.transaction.randomized_anchor_key.enabled`: Previously, concurrent transactions that constructed large write batches could cause [hotspots]({% link v24.2/ui-hot-ranges-page.md %}). This was because the [transaction record]({% link v24.2/architecture/transaction-layer.md %}#transaction-records) for all [ranges]({% link v24.2/architecture/reads-and-writes-overview.md %}#range) would coalesce on a single range, which would then cause this range's [leaseholder]({% link v24.2/architecture/reads-and-writes-overview.md %}#leaseholder) to perform all intent resolution work. This is fixed by distributing transaction records randomly across the ranges the write batch touches. In turn, hotspots are prevented. [#125744][#125744]
```

Comment:
This one includes the `.enabled` in the key, but the others haven't.

Reply:
ya these names were auto-pulled by a magic Python script from the generated cluster settings docs.  I'm not sure why some have `.enabled` and others don't.  In this case and the debug zip above I've removed the `.enabled` bit